env0/resource_environment.go

```diff
@@ -339,36 +339,60 @@ func resourceEnvironmentRead(ctx context.Context, d *schema.ResourceData, meta i
 
 	setEnvironmentSchema(d, environment, environmentConfigurationVariables)
 
+	if d.Get("template_id").(string) == "" {
+		// envrionment with no template.
+		template, err := apiClient.Template(environment.BlueprintId)
+		if err != nil {
+			return diag.Errorf("could not get template: %v", err)
+		}
+		if err := templateRead("without_template_settings", template, d); err != nil {
+			return diag.Errorf("schema resource data serialization failed: %v", err)
+		}
```

Comment:
about that part, for the environment "without template" we allow to edit minimal things in our FE,
here we expose it as part of the regular template - will it enable the user to edit extra stuff he can't from our FE?

Reply:
@eranelbaz - we had a discussion about. You can actually update all the fields in the UI.
See discussion here: https://env0.slack.com/archives/C01QBBEGBEH/p1660135769504129?thread_ts=1659988507.151169&cid=C01QBBEGBEH